composer/metrics/nlp.py

```diff
@@ -273,7 +281,7 @@ def rename_args(
             raise ValueError('Cannot use both `outputs` and `output_logits`')
         if output_logits is not None:
             warnings.warn(
-                VersionedDeprecationWarning('`output_logits` has been renamed to `outputs`.', remove_version='0.21.0'),
+                VersionedDeprecationWarning('`output_logits` has been renamed to `outputs`.', remove_version='0.23.0'),
```

Comment:
This change is already in the migration. Either we should update the code here in composer now (we're past this version) or just boot it to the migration (given that we don't wanna touch anything in the composer eval flow now).

Reply:
hm ok, yeah lets just remove this deprecation then, and it will go away when the whole thing goes away. sound good?

Reply:
That would remove the warning that the function signature will change with the migration? My instinct is to just change the deprecation version to tell ppl it'll change at the version that the migration is final.

Reply:
that works too